src/main/java/com/github/xetorthio/jedisque/BinaryJedisque.java

```diff
@@ -0,0 +1,62 @@
+package com.github.xetorthio.jedisque;
+
+import redis.clients.jedis.Protocol;
+import redis.clients.jedis.exceptions.JedisConnectionException;
+import redis.clients.util.SafeEncoder;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.*;
+
+public class BinaryJedisque extends redis.clients.jedis.Connection {
+	static final private int DISQUE_PORT = 7711;
+	private final List<URI> uris = new ArrayList<URI>();
+	private Random randomGenerator = new Random();
+
+	public BinaryJedisque() {
+		try {
+			uris.add(new URI("disque://localhost:" + DISQUE_PORT));
+		} catch (URISyntaxException e) {
+		}
+	}
+
+	public BinaryJedisque(URI... uris) {
+		this.uris.addAll(Arrays.asList(uris));
+	}
+
+	@Override
+	public void connect() {
+		while (!this.isConnected()) {
```

Comment:
I believe Jedisque will stop due to ``uris.remove`` in https://github.com/xetorthio/jedisque/pull/5/files#diff-8b337087bb392b880dd6a3b393c6ac92R41

It would be good, however, if the client wouldn't stop (i.e., hang looping forever) when one of the nodes comes back.